Data-Input-Methods.md

```diff
@@ -25,6 +34,17 @@ To define a new field, click Add next to the input or output field boxes. You
 will be presented with some or all of the fields below depending on whether you
 are adding an input or output field.
 
+###### Table 11-2. Field Description: Data Input Fields
+
+Name | Description
+--- | ---
+Field/Field Name | Either select or type in the name of the field you want to define. No spaces or other non-alphanumeric characters (except '-' or '_') should be used in this field.
```

Comment:
Does the rendered version of this look italicised as it does here in the diff?  If so, may need to add a \ before the '_' to escape it

Reply:
Testing...

Name | Description
--- | ---
Field/Field Name | Either select or type in the name of the field you want to define. No spaces or other non-alphanumeric characters (except '-' or '_') should be used in this field.

Reply:
Looks OK, so i'll merge 👍